src/ludamus/client/src/index.css

```
@@ -12,6 +12,8 @@
 @plugin "@tailwindcss/typography";
 @plugin "@hasparus/tailwind/scrollview-fade";
 
+@import "shadow-plugin";
```

Stylelint (17.14.1)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)